eq-author/src/App/QuestionnairesPage/QuestionnairesTable/Row.js

```diff
@@ -14,65 +15,81 @@ import QuestionnaireLink from "App/QuestionnairesPage/QuestionnaireLink";
 import FormattedDate from "App/QuestionnairesPage/FormattedDate";
 import FadeTransition from "components/transitions/FadeTransition";
 
+import { partial } from "lodash";
+import { rgba } from "polished";
+import { buildQuestionnairePath } from "utils/UrlUtils";
+
 const TruncatedQuestionnaireLink = Truncated.withComponent(QuestionnaireLink);
 TruncatedQuestionnaireLink.displayName = "TruncatedQuestionnaireLink";
 
+export const ShortTitle = styled.span`
+  color: ${colors.textLight};
+  text-decoration-color: ${colors.textLight};
+  font-size: 0.8em;
+  font-weight: bold;
+  text-transform: uppercase;
+  letter-spacing: 0.05em;
+`;
+
 const ButtonGroup = styled.div`
   display: flex;
-  justify-content: space-evenly;
 `;
 
-const TR = styled.tr`
-  border-bottom: 1px solid #e2e2e2;
+export const TR = styled.tr`
   border-top: 1px solid #e2e2e2;
+  border-bottom: 1px solid #e2e2e2;
+  background-color: ${props =>
+    props.odd ? "rgba(0,0,0,0)" : "rgba(0,0,0,0.02)"};
   opacity: 1;
   color: ${props => (props.disabled ? `${colors.textLight}` : "inherit")};
```

Comment:
You could wrap the whole property definition then you wouldn't need the inherit.
`${props => props.disabled && `color: ${colors.textLight};`}`

Reply:
I just realised that the prop disabled is never passed to this component, so the line can be removed.